controllers/orders/get-all/index.js

```diff
@@ -1 +1,22 @@
-export { getAllOrders } from './swagger.js';
+import { withPagination } from '../../../helpers/withPagination.js';
+import { ctrlWrapper } from '../../../middlewares/index.js';
+import Order from '../../../models/order/index.js';
+
+const controller = async (req, res) => {
```

Comment:
why do we need all orders from DB?

Reply:
We need this to show in the admin panel.